src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -134,6 +137,30 @@ import lendFilterPageQuery from '@/graphql/query/lendFilterPage.graphql';
 
 import lendFilterExpMixin from '@/plugins/lend-filter-page-exp-mixin';
 
+const lendFilterRedirectEXP = 'lend_filter_flss_v1';
+
+function isFLSSEligible(route = {}) {
+	// check route for eligibility
+	const eligibleQueryParams = ['page', 'sortBy', 'gender', 'registration'];
+	const queryParamKeys = Object.keys(route?.query);
+	const allowedSorts = ['expiringSoon', 'loanAmountDesc', 'loanAmount', 'popularity'];
+	// eligible by default, no params is also eligible
+	let isEligible = true;
+
+	queryParamKeys.forEach(queryParam => {
+		// ensure any query params are eligible
+		if (!eligibleQueryParams.includes(queryParam)) {
+			isEligible = false;
```

Comment:
Not a huge performance difference in this situation, but you could use a short circuited loop.